### PR TITLE
PR for #81: Model

### PR DESCRIPTION
**Pull request by santiagohermo**
Hey guys! Can you review the [model proposal](https://github.com/diegogentilepassaro/min_wage_rent/blob/253978213910c043cf1dc6124d95817156e1c679/issue81/model_proposal.pdf) built in #81? The code lives [here](https://github.com/diegogentilepassaro/min_wage_rent/blob/253978213910c043cf1dc6124d95817156e1c679/issue81/model_proposal.Rmd). I added a few things since the last time we talked. I think they need polishing, but illustrate other things we could do with this measure and how we could "sell" the paper.

Sorry for the delay! It's been a busy week. I think the goal of this PR is to polish this note and meet with professors to get their opinions on what to do with the paper. Diego and I have a presentation in the Applied Micro Lunch in two weeks, so maybe we can present it there as well to get some feedback.

Just for reference, 

- [here](https://github.com/diegogentilepassaro/min_wage_rent/blob/253978213910c043cf1dc6124d95817156e1c679/analysis/notes/model.Rmd) is the code for the more-complicated model I was working on earlier (pdf [here](https://github.com/diegogentilepassaro/min_wage_rent/blob/253978213910c043cf1dc6124d95817156e1c679/analysis/notes/model.pdf)), and 
- [here](https://www.overleaf.com/project/6011e2221cd9783d12536d10) is Diego's attempt.

### CODE REVIEW

File: issue81/model_proposal.Rmd [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+\frac{d y_{ni}}{d \underline{w}^L} > 0 , 
+\quad 
+\frac{d y_{ni}}{d \underline{w}^R} < 0
+\quad \quad  \forall (n,i) ,
+$$
+
+so that workplace minimum wages increase disposable income of these workers, whereas 
+residence ones lower it.
+
+We denote housing demand by the $(c,i)$ group as $H_{ni} \left(r_n, y_{ni} \right)$,
```
Comment (diegogentilepassaro) [inline issue81/model_proposal.Rmd:65]: $(c,i)$ should be $(n,i)$

**Review by diegogentilepassaro — APPROVED** [review]
Just found 1 typo! 

After the presentation, we can try a model in which we fix exogenously residence locations but with a transport cost, we allow for workplace location choice. That should be representative of the short term and should be enough to generate that the households from the same type chose different workplaces. We can have a firm that produces local good using both MW and non-MW workers and housing supply is fixed. 

That simple framework should make it easy to get welfare under fixed residence and fixed house supply. That sounds to me like a good way to start thinking about microfounding the model, instead of the old approach I tried where estimation was needed.

File: issue81/model_proposal.Rmd (hunk 1 above)
Comment (santiagohermo) [reply issue81/model_proposal.Rmd]: Fixed!

**Comment by santiagohermo** [comment]
Thanks @diegogentilepassaro! [That](https://github.com/diegogentilepassaro/min_wage_rent/pull/91#pullrequestreview-594659002) idea sounds very interesting.

Another doubt I have is whether we should call this argument:

https://github.com/diegogentilepassaro/min_wage_rent/blob/defd77d780c7aba67d18e0782f37a27be54a8b04/issue81/model_proposal.Rmd#L185-L190

an "identification" argument. I think for that we have to say that, in the version with time, you can't include both MW measures since they are collinear or something. Just something to think about.

I'll delete this branch then, and we can go back to the [code](https://github.com/diegogentilepassaro/min_wage_rent/blob/defd77d780c7aba67d18e0782f37a27be54a8b04/issue81/model_proposal.Rmd) or [pdf](https://github.com/diegogentilepassaro/min_wage_rent/blob/defd77d780c7aba67d18e0782f37a27be54a8b04/issue81/model_proposal.pdf) in the future when writing this down somewhere.

**Pull request closed without merging**